scripts/linters/pylint_extensions.py

```diff
@@ -1858,6 +1858,37 @@ def process_tokens(self, tokens):
                         line=line_num)
 
 
+class InequalityWithNoneChecker(checkers.BaseChecker):
+    """Custom pylint checker prohibiting use of "if x != None" and
+    enforcing use of "if x is not None" instead.
+    """
+
+    __implements__ = interfaces.IAstroidChecker
+
+    name = 'inequality-with-none'
+    priority = -1
+    msgs = {
+        'C0030': (
+            'Please refrain from using "x != None" '
+            'and use "x is not None" instead.',
+            'inequality-with-none',
+            'Use "is" to assert equality or inequality against None.'
+        )
+    }
+
+    def visit_compare(self, node):
+        """Called for comparisons (a != b).
+
+        Args:
+            node: astroid.node.compare. Node comparing two values.
+        """
+
+        ops = node.ops
+        for (operator, operand) in ops:
```

Comment:
remove unwanted parenthesis*
```suggestion
        for operator, operand in ops:
```

Reply:
Done!